src/test/java/org/sagebionetworks/web/unitclient/widget/entity/NbConvertPreviewWidgetTest.java

```diff
@@ -171,16 +181,40 @@ public void testFileHandleAssociation() {
   }
 
   @Test
-  public void testRequestBuilderConfigure() {
+  public void testRequestBuilderConfigureDev() {
+    previewWidget = setupPreviewWidget(REPO_URL_DEV);
+    previewWidget.configure(ENTITY_ID, mockFileHandle);
+    String actualNbConvertEndpoint = String.format(NBCONVERT_ENDPOINT, "dev");
```

Comment:
Indirectly testing your utility to java format() output for realistic input, I like it